pkg/importer/spanner/sysl.arrai

```diff
@@ -0,0 +1,138 @@
+# size returns the size of an attribute
+let size = \length
+    cond {
+        length = 'MAX': '',
+        //seq.has_prefix('0x', length): '(' ++ //seq.trim_prefix('0x', length) ++ ')',
+        length > 0: $`(${length})`,
+        _: '',
+    };
+
+# attributePatternToken returns the token based on attribute value
+let attributePatternToken = \entity \attr \token
+    cond {
+        (attr.options?:{}): token,
```

Comment:
Nit: these parens around the cond statements are unnecessary. Surprisingly many parens are unnecessary in a arr.ai, and it can often read more naturally without them.

```suggestion
        attr.options?:{}: token,
```

Reply:
ok